src/components/DataView/DataViewTHead/DataViewTHead.tsx

```diff
@@ -1,17 +1,16 @@
 import React from "react";
 import styled from "styled-components";
-// import jsvalidator from "jsvalidator";
 
 import SwitchLeftIcon from "@mui/icons-material/SwitchLeft";
 import SwitchRightIcon from "@mui/icons-material/SwitchRight";
 
 import Checkbox from "@root/components/Checkbox";
-import DataViewBulkActionsButtonsRow from "../DataView/DataViewBulkActionsButtonsRow";
-import DataViewBulkAllBar from "../DataView/DataViewBulkAllBar";
+import DataViewBulkActionsButtonsRow from "../DataViewBulkActionsButtonsRow/DataViewBulkActionsButtonsRow";

```

Comment:
This import can also be reduced to `import DataViewBulkActionsButtonsRow from "../DataViewBulkActionsButtonsRow";`